mixins/property-required/property-required-mixin.js

```diff
@@ -6,19 +6,14 @@ export function createDefaultMessage(tagName, propertyName) {
 	return `${tagName}: "${propertyName}" attribute is required.`;
 }
 
-export function createUndefinedPropertyMessage(tagName, propertyName) {
-	return `PropertyRequiredMixin: "${tagName.toLowerCase()}" has no property "${propertyName}".`;
-}
```

Comment:
Bonus: requiring a property that isn't defined is no longer a possibility, so this error case goes away.